source/Core/BSP/Sequre_S60/Software_I2C.h

```diff
@@ -10,6 +10,7 @@
 #include "BSP.h"
 #include "configuration.h"
 #include "stm32f1xx_hal.h"
+#define I2C_SOFT_BUS_2

```

Comment:
This should not be here, this should be in `configuration.h`

Reply:
oh yeah forgot to remove it